src/Console/Command/FixCommand.php

```diff
@@ -112,6 +112,7 @@ protected function configure(): void
                     new InputOption('format', '', InputOption::VALUE_REQUIRED, 'To output results in other formats.'),
                     new InputOption('stop-on-violation', '', InputOption::VALUE_NONE, 'Stop execution on first violation.'),
                     new InputOption('show-progress', '', InputOption::VALUE_REQUIRED, 'Type of progress indicator (none, dots).'),
+                    new InputOption('diff-format', '', InputOption::VALUE_OPTIONAL, 'Ignored option. Always equals to "udiff".'),
```

Comment:
it sounds like a tech-debt that you ask to introduce to 3.0.0, while 3.0.0 purpose is exactly to drop the legacy tech-debt we have, diff-format was one if it. let's discuss in #5600